### PR TITLE
Document data export process.

### LINKED ISSUE #184: Document process for exporting CMS database
Exporting data from the CMS database is kind of a pain, since we don't have direct access to the database. I'll refresh my memory on the process, then demo to @LindsayYoung and/or @ccostino and document how it works.

### PR DESCRIPTION
**Pull request by jmcarp**
[Resolves #184]

To review: @LindsayYoung and/or @ccostino, try out the instructions in `README.rst`, and revise or ping me if you run into issues or find anything unclear.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -31,10 +31,10 @@ We’re thrilled you want to get involved!
 
 
 Set up
-============
+======
 
 Installation
------------------
+------------
```
Comment (ccostino) [inline README.rst]: Is there anything else that needs to be added to the PostgreSQL setup, such as creating a database, the name associated with it, and any users?

Comment (jmcarp) [inline README.rst]: No, we haven't changed the default of using sqlite for development, so there's no setup to do here. Setting up databases in cloud.gov is a one-liner that's documented under "Deploy".

**Comment by ccostino** [comment]
@jmcarp:  One thing I noticed (thought it's not with your changes) are the steps involved with running the few `./manage.py` commands are a bit out of order.  The `createsuperuser` step should come after the two migration related steps.

**Comment by LindsayYoung** [comment]
I really think followed the instructions exactly with `--exclude auth.permission` But I am seeing the fec super user account and password, I think we need `--exclude auth.user` too.

Got most of this working, I do get an error when I try to load, I think it was because I already added a calendar page.

Just wanted to note that the conflict is easy and I haven't forgotten this.

**Pull request merged**